core/src/components/datetime/datetime.ios.scss

```diff
@@ -267,6 +267,11 @@
   color: current-color(contrast);
 }
 
+:host .calendar-day.calendar-day-hidden-day:not([disabled])  {
+  opacity: 1;
```

Comment:
Maybe the opacity: 1 rule can be moved to the common styles

Reply:
but if we're overriding both in md and ios we can move it to the `datetime.native.scss` I believe